Java/avg/array.java

```diff
@@ -12,4 +12,15 @@ public static void main(String[] args) {
             System.out.println(arr[i]);
         }
     }
+
+    public static void Even(String[] args) {
+        Scanner sc = new Scanner(System.in);
+        int[] arr = new int[]{1, 2, 3, 4, 5};
+        System.out.println("Even numbers:");
+
+        for (int i = 0; i < arr.length; i = i * 2)
+        {
+            System.out.println(arr[i]);
+        }
+    }

```

Comment:
**Suggestion**: Consider abstracting the logic for printing elements of the array into a separate method that accepts the starting index and the increment as parameters. This would reduce code duplication and improve maintainability. <br> **Code Suggestion**: 
 ```
 +    public static void printArrayElements(int[] arr, int start, int increment) {
 +        for (int i = start; i < arr.length; i += increment) {
 +            System.out.println(arr[i]);
 +        }
 +    }
 +    public static void Even(String[] args) {
 +        int[] arr = new int[]{1, 2, 3, 4, 5};
 +        printArrayElements(arr, 1, 2); // For even
 +    }
 ```